app/scripts/metamask-controller.js

```diff
@@ -636,15 +709,19 @@ module.exports = class MetamaskController extends EventEmitter {
         case 'signed':
           return cb(null, data.rawSig)
         case 'rejected':
-          return cb(new Error('MetaMask Message Signature: User denied transaction signature.'))
+          return cb(new Error('MetaMask Message Signature: User denied message signature.'))
         default:
           return cb(new Error(`MetaMask Message Signature: Unknown problem: ${JSON.stringify(msgParams)}`))
       }
     })
   }
 
   /**
-   * @param  {} msgParams
+   * Signifies a user's approval to sign a personal_sign message in queue.
+   * Triggers signing, and the callback function from newUnsignedPersonalMessage.
+   *
+   * @param {Object} msgParams - The params of the message to sign & return to the Dapp.
+   * @returns {Promise<Object} - A full state update.
```

Comment:
This type is missing a closing `>` on the `Promise` return type of `Object`.

Reply:
A lot of this is Doc linting. Is there a JSDoc linter we can easily run or add to our ESLintrc? I have my linter run before every `git push`, exactly because I hate burdening people with this kind of minutiae.